.gitignore

```diff
@@ -81,3 +81,6 @@ go.work.sum
 # Quickstart helpers
 shannon_appgate_config.yaml
 shannon_app_config.yaml
+
+# localnet_config
+localnet_config.yaml-e
```

Comment:
Is the trailing `-e` intentional?

Reply:
[`01ed1ff` (#298)](https://github.com/pokt-network/poktroll/pull/298/commits/01ed1ff1f2584430faad2c6faf58e11b91dd08f2)